svthalia/concrexit
# Add VSCode settings

Closes #3371.

This checks in:

- Recommended extensions for linting and formatting
- Settings for those extensions
- A launch config for manage.py runserver and test with debugger 

### How to test
Try working on concrexit on a laptop with only VSCode, and no python-specific configuration. The see that, if you follow the recommendations, you get format on save and the rigt linting out of the box.
